privacyidea/api/auth.py

```diff
@@ -339,6 +339,11 @@ def get_auth_token():
                                     "serial": serials,
                                     "info": "{0!s}|loginmode={1!s}".format(log_used_user(user_obj),
                                                                            details.get("loginmode"))})
+            if "multi_challenge" in details:
```

Comment:
We also need to check if `user_auth` is `True`

Reply:
fixed in 2a389e89b39bb6c60c8372b339772a398ca7898a